spec/metric_fu/metrics/rails_best_practices/configuration_spec.rb

```diff
@@ -20,9 +20,23 @@
         end
       end
 
-      it "should set @rails_best_practices to {}" do
+      it "should default @rails_best_practices to { :silent => true }" do
         load_metric "rails_best_practices"
-        expect(MetricFu::Metric.get_metric(:rails_best_practices).run_options).to eql({})
+        expect(MetricFu::Metric.get_metric(:rails_best_practices).run_options).to eq(
+                                                                                    exclude: [],
+                                                                                    silent: true
+                                                                                  )
+      end
+
+      it "can configure @rails_best_practices 'exclude' using the sugar" do
+        load_metric "rails_best_practices"
+        rbp = MetricFu::Metric.get_metric(:rails_best_practices)
+        rbp.exclude = ["config/chef"]
```

Comment:
Without looking at the code, I don't recall. It should be reconfiguring / reseting metrics, but maybe that's not happening here.  Do you want to pair on it or just get something simple working with a TODO note to do it right, so that I can merge it?